package.json

```diff
@@ -27,25 +27,24 @@
     "database"
   ],
   "devDependencies": {
-    "eslint": "^6.8.0",
-    "eslint-plugin-require-path-exists": "^1.1.9",
-    "jsdoc": "^3.5.5",
-    "docdash": "^1.0.3",
-    "mocha": "^6.0.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "lodash": "^4.17.11",
+    "compare-versions": "^3.6.0",
```

Comment:
What are the changes here in the devDependencies section?

Reply:
They were generated. I'll remove this and other changes not related to the session code and we can look at them separately